nservicebus/azure-storage-persistence/index.md

```diff
@@ -18,9 +18,9 @@ Certain features of NServiceBus require persistence to permanently store data. A
 
 ## How to enable persistence with Azure Storage Services
 
-First add a reference to the assembly that contains the Azure storage persisters. The recommended way of doing this is by adding a NuGet package reference to the `NServiceBus.Azure` package.
+First add a reference to the assembly that contains the Azure storage persisters. When working with NServiceBus Version 5 or lower this is done by adding a NuGet package reference to `NServiceBus.Azure`. For NServiceBus Version 6 and above the NuGet package reference will be to `NServiceBus.Persistence.AzureStorage`.
 
-If self hosting, the persistence technology could be configured using the configuration API and the extension method found in the `NServiceBus.Azure` assembly.
+If self hosting, the persistence technology should be configured using the configuration API.
```

Comment:
Should? What parameters do I use to make that decision? I'd make it more forceful:

> If self hosting, configure the persistence technology using the configuration API.